### PR TITLE
Todo list interractions

### PR DESCRIPTION
**Pull request by kennyegun24**
# In this pr, I added the following functionalities

- Added confirmation to checkbox values.
- Implemented the remove all button for both UI and local storage

### CODE REVIEW

File: modules/clearUi.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/* eslint-disable*/
+const input = document.querySelector('.input');
+
+const clear = () => {
+  input.value = '';
+};
+
```
Comment (bobb-Rob) [inline modules/clearUi.js]: - [ ] [REQUIRED] Welldone so far but I notice a few things:
    - Please ensure that the page reloads and updates after any interaction applied to your program. Eg. the checkbox does not update in the local storage until I manually refresh. 
    - Secondly, after completed is marked true, if I reload the page, the UI does reflect the completed status initially applied. 

Please check your code again and ensure that interactions are stored and loaded correctly on reload. in other words, the updated array of items is stored in local storage, so the user gets the correct list values after the page reloads.

File: modules/clearUi.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/* eslint-disable*/
+const input = document.querySelector('.input');
+
+const clear = () => {
+  input.value = '';
+};
```
Comment (bobb-Rob) [inline modules/clearUi.js]: This module folder should be in the src folder. Kindly fix this.

File: modules/clearUi.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+/* eslint-disable*/
+const input = document.querySelector('.input');
+
+const clear = () => {
+  input.value = '';
```
Comment (bobb-Rob) [inline modules/clearUi.js]: - [ ] [REQUIRED] This file `clearUi.js` should contain methods related to the status updates (`completed`: `true` / `false`).
- [ ] It has an event listener on the checkbox (`change`). 
- [ ] This file should contain the function for the "Clear all completed" button using the `Array.filter()` method.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @kennyegun24 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mgmediaweb — APPROVED** [review]
Hi @kennyegun24 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved6

## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**